.gitmodules

```diff
@@ -62,3 +62,6 @@
 [submodule "src/redis-dump-load"]
 	path = src/redis-dump-load
 	url = https://github.com/p/redis-dump-load.git
+[submodule "platform/broadcom/sonic-platform-modules-netberg"]
```

Comment:
we no longer accept submodule, please have source code checked in in the buildimage rep.